src/MeadeCommandProcessor.cpp

```diff
@@ -816,22 +816,30 @@ bool gpsAqcuisitionComplete(int &indicator);  // defined in c72_menuHA_GPS.hpp
 //      Returns:
 //        "HHMMSS"
 //
-// :XGh#
+// :XGhN#
 //      Description:
-//        Get Heater values
+//        Get value for heaters
 //      Information:
-//        Get the current values for both heaters
+//        Get value for heater N in the range from 0-10, or both heaters if no N is given.
```

Comment:
If you want to support more than 2 heaters, you'll need to adapt the EEPROM code accordingly. Also this needs to be limited to 9, since otherwise the setter can't figure out whether it's one or two digits (currently it checks for one only)

Reply:
Oh, ok. The wording is ambiguous.
`Get value for heater N (either 0 or 1) or both if no N is given. Returned value is in the range 0-10.`